lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -121,8 +121,11 @@ function dosomething_metatag_get_permalink_vars() {
   $rbid = arg(1);
   $fid = ($_GET['fid']);
   $rb_entity = reportback_load($rbid);
-  if (!$fid) {
-    $fid = $rb_entity->fids[0];
+  if (!$fid && !empty($rb_entity->fids)) {
+    $fid = array_pop($rb_entity->fids);
+  }
+  else {
+    $fid = array_pop($rb_entity->getFids());
```

Comment:
It seems at least a more readable fix could be to always `array_pop($rb_entity->getFids());` and avoid the `if`/`else`... but it would def be nice to have this in the one singular place, whether it be a property on a Reportback or a function you always need to call on a loaded Reportback, and to not really chalk this one up to "who knows whats going on"..